packages/schematics/src/utility/setup-project.ts

```diff
@@ -0,0 +1,25 @@
+import { UnitTestTree, SchematicTestRunner } from '@angular-devkit/schematics/testing';
+
+export async function setupProject(tree: UnitTestTree, schematicRunner: SchematicTestRunner, name: string) {
+  tree = await schematicRunner
+    .runExternalSchematicAsync('@schematics/angular', 'workspace', {
+      name: 'workspace',
+      version: '10.0.0',
```

Comment:
It looks like we are hard coding the Angular version here. Does that mean it will need to be updated each time we upgrade the Anger version?

Reply:
I think we won't have to bump the version unless there are drastic changes in the  Angular project structure. Regardless, I'll check for other options.

Reply:
I've just checked the Angular schematics tests. The version there's hardcoded to `6.0`.
https://github.com/angular/angular-cli/blob/master/packages/schematics/angular/component/index_spec.ts#L38
https://github.com/angular/angular-cli/blob/master/packages/schematics/angular/service/index_spec.ts#L27